runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -215,13 +218,24 @@ void rebuild()
 			priceTypeString = priceType == LootTrackerPriceType.HIGH_ALCHEMY ? "HA: " : "GE: ";
 		}
 
-		priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
-		priceLabel.setToolTipText(QuantityFormatter.formatNumber(totalPrice) + " gp");
+		String averageString = kills > 1 ? QuantityFormatter.formatNumber(totalPrice / kills) + " gp" : "0";
+		String totalString = priceTypeString + QuantityFormatter.formatNumber(totalPrice) + " gp";
+		if (showAverage)
+		{
+			priceLabel.setText(averageString);
+			priceLabel.setToolTipText(totalString + " (total)");
+		}
+		else
+		{
+			priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice));
```

Comment:
Compared to the price label text prior to this change, this is missing the `+ " gp"` suffix. `totalString` should be used instead
```suggestion
			priceLabel.setText(totalString);
```

Reply:
I can't believe I missed that sorry.  I've pushed up a change that removes the totalString, in favour of making it exactly the same as before.

Reply:
@Nightfirecat  - I think this conversation can be resolved